npm/webpack-dev-server/src/loader.ts

```diff
@@ -59,22 +61,33 @@ export default function loader (this: unknown) {
   ctx.cacheable(false)
   const { files, projectRoot, supportFile } = ctx._cypress
 
-  const supportFileAbsolutePath = supportFile ? JSON.stringify(path.resolve(projectRoot, supportFile)) : undefined
-
-  return `
-  var loadSupportFile = ${supportFile ? `() => import(${supportFileAbsolutePath})` : `() => Promise.resolve()`}
+  const supportFileAbsolutePath = supportFile ? path.resolve(projectRoot, supportFile) : undefined
```

Comment:
Good question - this came out of the spike so I may as @ZachJW34 to chime in - happen to remember why this change was made?

Reply:
I thought the `JSON.Stringify` simply wrapped the string in quotes, so rather than do `load: () => import("${thing}")` you can do `load: () => import(${JSON.stringify(thing)})`. I'd be fine reverting back to using `JSON.stringify`, just in case it's doing more than what I thought.

Reply:
![image](https://user-images.githubusercontent.com/19196536/191903125-ceaa104b-69e4-4e54-bf00-64af0e3c3e96.png)

Seems to add backslashes, at least in the console - I'm pretty sure this was here for a specific reason, might be good to revert it until we know it's safe to remove.